packages/polymath-issuer/src/utils/parsers/index.js

```diff
@@ -72,6 +72,7 @@ export function parseWhitelistCsv(file: string) {
   const data = csvParse(file, {
     skip_empty_lines: true,
     trim: true,
+    relax_column_count: true,
```

Comment:
Why do we need to use this?

Reply:
This was initially used to handle the inconsistent number of fields between the header and data rows